helm/files/uptime.yaml

```
@@ -0,0 +1,28 @@
+        severity: critical
+      annotations:
+        summary: The Validator's uptime is decreasing now! Check your validator in {{ $labels.chain }}-{{ $labels.chain_id }} network
+
+    - alert: Remaining1000BlocksBeforeGettingSlashing
+      expr: |
+        # max missed - current missed 
+        ceil(cvms_uptime_signed_blocks_window * (1 - cvms_uptime_min_signed_per_window)) - on (chain_id) group_right () cvms_uptime_missed_blocks_counter < 1000
+      labels:
+        severity: critical
+      annotations:
+        summary: The Validator's is missing too many! Check your validator in {{ $labels.chain }}-{{ $labels.chain_id }} network
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)